libs/build_ecbuild.sh

```diff
@@ -35,11 +35,20 @@ URL="https://github.com/$repo/$name.git"
 [[ -d $software ]] && cd $software || ( echo "$software does not exist, ABORT!"; exit 1 )
 
 git checkout $version
+
+# Conform to NCO IT FISMA High Standards
+if [[ ${NCO_IT_CONFORMING:-"NO"} =~ [yYtT] ]]; then
+  sed -r -i 's|ssh://[a-zA-Z0-9.@]*||g' cmake/compat/ecmwf_git.cmake
+  sed -r -i 's|http[]://[a-zA-Z0-9@${}_./]*||g' cmake/compat/ecmwf_git.cmake
+  sed -r -i 's|http[]://[a-zA-Z0-9./-]*/test-data|DISABLED_BY_DEFAULT|g' cmake/ecbuild_check_urls.cmake
+  sed -r -i 's|http[]://[a-zA-Z0-9./-]*/test-data|DISABLED_BY_DEFAULT|g' cmake/ecbuild_get_test_data.cmake
+fi
+
 [[ ${DOWNLOAD_ONLY} =~ [yYtT] ]] && exit 0
 [[ -d build ]] && $SUDO rm -rf build
 mkdir -p build && cd build
 
-cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
+../bin/ecbuild --prefix=$prefix ..
```

Comment:
Is there a particular reason why `ecbuild` is used here, and not `cmake`?

Reply:
Just following the installation directions provided by the developers.
https://github.com/ecmwf/ecbuild/blob/develop/INSTALL.rst

Reply:
Ok, fair enough. I think that spack-stack always builds with plain cmake, but as long as the ecbuild include/macro loads are up first it works the same way.